Library/UIViewController-Preparation.swift

```diff
@@ -43,16 +44,21 @@ extension UIViewController {
     self.bindViewModel()
   }
 
+  internal func ksr_viewWillAppear(_ animated: Bool) {
+    self.ksr_viewWillAppear(animated)
+    self.bindStyles()
+  }
```

Comment:
@stephencelis remember that weird out of order bindstyles thing I was telling you about? well i couldnt figure out why it was happening, so i decided to just do styling in viewwillappear too :/

not ideal, but couldnt figure it out. maybe we could look at it together when im back from cape town